modules/picard_qc/1.0/config/default.yaml

```diff
@@ -0,0 +1,58 @@
+lcr-modules:
+    
+    picard_qc:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "__UPDATE__"
+            intervals : "__UPDATE__"
+
+            # by default are generate by reference_files subworkflow
+            rrna_intervals: "genomes/{genome_build}/rrna_intervals/rRNA_int_gencode-33.txt"
+            refFlat: "genomes/{genome_build}/annotations/refFlat_gencode-33.txt"
+
+        options:
+            alignment_summary: "VALIDATION_STRINGENCY=LENIENT"
+            insert_size: "VALIDATION_STRINGENCY=LENIENT"
+            hs_metrics: "VALIDATION_STRINGENCY=LENIENT LEVEL=ALL_READS CLIP_OVERLAPPING_READS=false"
+            rnaseq_metrics:
+                base: "VALIDATION_STRINGENCY=LENIENT"
+                strand:
+                    _default: "NONE"
+                    forward: "FIRST_READ_TRANSCRIPTION_STRAND"
+                    reverse: "SECOND_READ_TRANSCRIPTION_STRAND" 
```

Comment:
You can put this under `switches` like I did in `manta` 2.1. Not a required change though. Just a suggestion for consistency.

Reply:
so as a general rule should I put anything the require the `switch_on` functions under switches?